tools/releaser.sh

```diff
@@ -0,0 +1,202 @@
+  echo "  --dry-run - Do not create the release."
+  echo "  --sync-operator-repository <repository> - The operator repository to sync."
+}
+
+function parse_args() {
+  while [ "$#" -gt 0 ]; do
+    case "$1" in
+    --gh-org)
+      GH_ORG="$2"
+      shift
+      ;;
+    --gh-repo)
+      GH_REPO="$2"
+      shift
+      ;;
+    --oci-org)
+      OCI_ORG="$2"
+      shift
+      ;;
+    --dry-run)
+      DRY_RUN="true"
+      ;;
+    --sync-operator-repository)
+      SYNC_OPERATOR_REPOSITORY="$2"
+      shift
+      ;;
+    -h | --help)
+      usage
+      exit 0
+      ;;
+    *)
+      usage
+      exit 1
+      ;;
+    esac
+    shift
+  done
+}
+
+function strip_prefix() {
+  local str="$1"
+  local prefix="$2"
+
+  echo ${str#"$prefix"}
+}
+
+function version_without_v() {
+  strip_prefix "$1" v
+}
+
+function version_with_v() {
+  local without_v=$(version_without_v "$1")
+
+  echo "v$without_v"
+}
+
+function get_noobaa_version() {
+  local version=$(npm -s run env echo '$npm_package_version')
+  if [[ $# -eq 1 ]]; then
+    if [[ "$1" == "1" ]]; then
+      version_with_v "$version"
+    else
+      version_without_v "$version"
+    fi
+  else
+    version_without_v "$version"
+  fi
+}
+
+# bump_semver_patch takes in a semver and bumps the patch version
+function bump_semver_patch() {
+  local version=$(version_without_v "$1")
+  version=$(echo ${version} | awk -F. -v OFS=. '{$NF = $NF+1 ; print}')
+  echo "$version"
+}
+
+function create_noobaa_image() {
+  echo "Creating noobaa image"
+  make noobaa
+
+  echo "Tagging noobaa image"
+  docker tag noobaa ${OCI_ORG}/noobaa-core:$(get_noobaa_version)
+}
+
+function create_oci_release() {
+  # Release OCI images to docker and quay.io
+  if [[ "$DRY_RUN" == "true" ]]; then
+    echo "DRY_RUN is set, skipping OCI release creation."
+    return
+  fi
+
+  local quay_image="quay.io/$QUAY_USERNAME/noobaa-core:$(get_noobaa_version)"
+  local docker_image="$DOCKERHUB_USERNAME/noobaa-core:$(get_noobaa_version)"
+
+  echo "Tagging the images..."
+  docker tag "$OCI_ORG/noobaa-core:$(get_noobaa_version)" $quay_image
+  docker tag "$OCI_ORG/noobaa-core:$(get_noobaa_version)" $docker_image
+
+  echo "Logging in to docker.io..."
+  echo "$DOCKERHUB_TOKEN" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+
+  echo "Pushing the images to docker.io..."
+  docker push $docker_image
+
+  echo "Logging in to quay.io..."
+  echo "$QUAY_TOKEN" | docker login -u "$QUAY_USERNAME" --password-stdin quay.io
+
+  echo "Pushing the images to quay.io..."
+  docker push $quay_image
+}
+
+function sync_operator() {
+  if [[ -z "$SYNC_OPERATOR_REPOSITORY" ]]; then
+    echo "Skip Syncing operator, no repository was provided."
+    return
+  fi
+
+  gh workflow run releaser.yaml -R "$GH_ORG/$SYNC_OPERATOR_REPOSITORY" -f base_branch="$BASE_BRANCH"
+}
+
+function update_package_json() {
+  local version=$(bump_semver_patch $(get_noobaa_version))
+  npm version ${version} --no-git-tag-version
```

Comment:
Is it also updating the package-lock file?

Reply:
Yes, I checked that running `npm version <version> --no-git-tag-version` not only updates the version in `package.json` but also updates the `package-lock.json`.